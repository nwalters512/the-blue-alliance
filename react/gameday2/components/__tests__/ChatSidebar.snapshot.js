import React from 'react'
import renderer from 'react-test-renderer'
import ChatSidebar from '../ChatSidebar'

describe('ChatSidebar snapshot', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <ChatSidebar />
    ).toJSON()
    expect(component).toMatchSnapshot()
  })

  it('renders correctly when hidden', () => {
    const component = renderer.create(
      <ChatSidebar hidden />
    ).toJSON()
    expect(component).toMatchSnapshot()
  })
})
