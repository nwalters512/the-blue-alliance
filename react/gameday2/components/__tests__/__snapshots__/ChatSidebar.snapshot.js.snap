exports[`ChatSidebar snapshot renders correctly 1`] = `
<div
  className="chat-sidebar">
  <iframe
    frameBorder="0"
    height="100%"
    id="chat_embed"
    scrolling="no"
    src="http://twitch.tv/chat/embed?channel=tbagameday&popout_chat=true"
    width="100%" />
</div>
`;

exports[`ChatSidebar snapshot renders correctly when hidden 1`] = `
<div
  className="chat-sidebar">
  <iframe
    frameBorder="0"
    height="100%"
    id="chat_embed"
    scrolling="no"
    src="http://twitch.tv/chat/embed?channel=tbagameday&popout_chat=true"
    width="100%" />
</div>
`;
